nomad/client_csi_endpoint_test.go

```diff
@@ -548,17 +512,14 @@ func setupLocal(t *testing.T) (rpc.ClientCodec, func()) {
 	mockCSI.NextDeleteSnapshotError = fmt.Errorf("no plugins registered for type")
 	mockCSI.NextListExternalSnapshotsError = fmt.Errorf("no plugins registered for type")
 
-	c1, cleanupC1 := client.TestClientWithRPCs(t,
-		func(c *config.Config) {
-			c.Servers = []string{s1.config.RPCAddr.String()}
-		},
+	c1, cleanupC1, err := client.TestRPCOnlyClient(t, nil, s1.config.RPCAddr,
```

Comment:
just flagging that this is the signal in the require->must noise (which is also good!)